zebra/zebra_vty.c

```diff
@@ -183,7 +376,9 @@ static int zebra_static_route_leak(struct vty *vty,
 		if (strncasecmp(ifname, "Null0", strlen(ifname)) == 0 ||
 		    strncasecmp(ifname, "reject", strlen(ifname)) == 0 ||
 		    strncasecmp(ifname, "blackhole", strlen(ifname)) == 0) {
-			vty_out(vty, "%% Nexthop interface cannot be Null0, reject or blackhole\n");
+			if (vty)
+				vty_out(vty,
+					"%% Nexthop interface cannot be Null0, reject or blackhole\n");
 			return CMD_WARNING_CONFIG_FAILED;
```

Comment:
that event should be logged into a zlog_xxx

Reply:
done